clang_12/Dockerfile

```diff
@@ -0,0 +1,97 @@
+FROM ubuntu:bionic
+
+LABEL maintainer="Luis Martinez de Bartolome <luism@jfrog.com>"
```

Comment:
Since I copy pasted the clang_11 Dockerfile, I left this line with the maintainer also intact. Please let me know if this is fine.